pages/attacks/Cross-site-Scripting-(XSS).md

```diff
@@ -66,7 +66,7 @@ test for the various kinds of XSS vulnerabilities.
 
   - [Testing_for_Reflected_Cross_site_scripting](https://github.com/OWASP/wstg/blob/master/document/4-Web_Application_Security_Testing/07-Input_Validation_Testing/01-Testing_for_Reflected_Cross_Site_Scripting.md)
   - [Testing_for_Stored_Cross_site_scripting](https://github.com/OWASP/wstg/blob/master/document/4-Web_Application_Security_Testing/07-Input_Validation_Testing/02-Testing_for_Stored_Cross_Site_Scripting.md)
-  - [Testing_for_DOM-based_Cross_site_scripting](https://github.com/OWASP/wstg/blob/master/document/4-Web_Application_Security_Testing/11-Client_Side_Testing/01-Testing_for_DOM-based_Cross_Site_Scripting.md)
+  - [Testing_for_DOM-based_Cross_site_scripting](https://github.com/OWASP/wstg/blob/master/document/4-Web_Application_Security_Testing/11-Client-side_Testing/01-Testing_for_DOM-based_Cross_Site_Scripting.md)
```

Comment:
https://owasp.org/www-project-web-security-testing-guide/#how-to-reference-wstg-scenarios
```suggestion
  - [Testing_for_DOM-based_Cross_site_scripting](https://owasp.org/www-project-web-security-testing-guide/v41/4-Web_Application_Security_Testing/11-Client_Side_Testing/01-Testing_for_DOM-based_Cross_Site_Scripting.html)
```